modules/kernel/src/org/apache/axis2/Constants.java

```diff
@@ -84,6 +84,10 @@ public class Constants extends org.apache.axis2.namespace.Constants {
      * To change the service path to something else
      */
     public static final String PARAM_SERVICE_PATH = "servicePath";
+    /**
+     * Parameter to enable strict service path validation
+     */
+    public static final String PARAM_SERVICE_PATH_VALIDATION = "servicePathValidation";
```

Comment:
should we use strictServicePathValidation ? WDYT?

Reply:
+1 fixed